src/main/webapp/WEB-INF/tags/student/comments/feedbackSessionPanel.tag

```diff
@@ -6,8 +6,11 @@
 <div class="panel panel-primary">
     <div class="panel-heading">
         <strong>Comments in session: ${feedbackSessionRow.feedbackSessionName}</strong>
+        <div class='display-icon pull-right'>
```

Comment:
I do added collapse/ expand functionality to comments Page of Student . Am I supposed to do this only Instructor Search Page for this issue ?